### DISCUSSION TITLE
How to delete vertices and edges?

### DISCUSSION THREAD

**Discussion by riverfr0zen** [arun1729/cog#33] [General]
Is there a way to drop vertices and/or edges from the graph? For example, deleting a vertex and also removing all incoming and outgoing edges. I see there is a low-level method to delete from the key-val store, but not sure how to use it for this purpose.

**Comment by arun1729** [comment]
yes, currently there is no way to do it. The low level API is there as a foundation for future delete functions. It is on the road map and the next release will have at least some delete function.

**Comment by arun1729** [comment]
Feature to delete an edge (a triple) has been added a while ago.
